docs: Added new getting started page (#232)
apify/apify-client-python: docs/index.md

and returns it in the expected format.
Date strings are automatically converted to `datetime.datetime` objects.
For exceptions, we throw an [`ApifyApiError`](/reference/class/ApifyApiError),
which wraps the plain JSON errors returned by API and enriches them with other context for easier debugging.

```python
from apify_client import ApifyClient

apify_client = ApifyClient('MY-APIFY-TOKEN')

try:
    dataset_items = apify_client.dataset('not-existing-dataset-id').list_items().items
except Exception as ApifyApiError:
    # The exception is an instance of ApifyApiError
    print(ApifyApiError)
```

### Retries with exponential backoff

Network communication sometimes fails.
The client will automatically retry requests that failed due to a network error,
an internal error of the Apify API (HTTP 500+) or rate limit error (HTTP 429).
By default, it will retry up to 8 times.
First retry will be attempted after ~500ms, second after ~1000ms and so on.
You can configure those parameters using the `max_retries` and `min_delay_between_retries_millis` options
of the [`ApifyClient`](/reference/class/ApifyClient) constructor.

```python
from apify_client import ApifyClient

> apify_client = ApifyClient(
>     token='MY-APIFY-TOKEN',
>     max_retries=8,
>     min_delay_between_retries_millis=500, # 0.5s
>     timeout_secs=360, # 6 mins
>     )

Reviewer: fix indentation

```suggestion
apify_client = ApifyClient(
    token='MY-APIFY-TOKEN',
    max_retries=8,
    min_delay_between_retries_millis=500, # 0.5s
    timeout_secs=360, # 6 mins
)
```

```python
from apify_client import ApifyClient

apify_client = ApifyClient('MY-APIFY-TOKEN')

# Start an Actor and waits for it to finish
finished_actor_run = apify_client.actor('username/actor-name').call()

# Starts an Actor and waits maximum 60s (1 minute) for the finish 
actor_run = apify_client.actor('username/actor-name').start(wait_for_finish=60)
```

### Pagination

Most methods named `list` or `list_something` return a [`ListPage`](/reference/class/ListPage) object,
containing properties `items`, `total`, `offset`, `count` and `limit`.
There are some exceptions though, like `list_keys` or `list_head` which paginate differently.
The results you're looking for are always stored under `items` and you can use the `limit`
property to get only a subset of results. Other properties can be available depending on the method.

```python
from apify_client import ApifyClient

apify_client = ApifyClient('MY-APIFY-TOKEN')

# Resource clients accept an ID of the resource.
dataset_client = apify_client.dataset('dataset-id')

# Number of items per page
limit = 1000
# Initial offset
offset = 0
# List to store all items
all_items = []

> while True:
>     response = dataset_client.list_items(limit=limit, offset=offset)
>     items = response.items
>     total = response.total
> 
>     print(f"Fetched {len(items)} items")
> 
>     # Merge new items with other already loaded items
>     all_items.extend(items)
> 
>     # If there are no more items to fetch, exit the loading
>     if offset + limit >= total:
>         break
> 
>     offset += limit
> 
> print(f"Overall fetched {len(all_items)} items")
> 
> ```

Reviewer: unify quotes

```suggestion
while True:
    response = dataset_client.list_items(limit=limit, offset=offset)
    items = response.items
    total = response.total
    
    print(f'Fetched {len(items)} items')

    # Merge new items with other already loaded items
    all_items.extend(items)

    # If there are no more items to fetch, exit the loading
    if offset + limit >= total:
        break

    offset += limit

print(f'Overall fetched {len(all_items)} items')
```

[Key-value store](https://docs.apify.com/platform/storage/key-value-store) records can be retrieved as objects, buffers or streams via the respective options,
dataset items can be fetched as individual objects or serialized data, or iterated asynchronously.

```python
from apify_client import ApifyClient

apify_client = ApifyClient('MY-APIFY-TOKEN')

# Start an Actor and waits for it to finish
finished_actor_run = apify_client.actor('username/actor-name').call()

# Starts an Actor and waits maximum 60s (1 minute) for the finish 
actor_run = apify_client.actor('username/actor-name').start(wait_for_finish=60)
```

### Pagination

Most methods named `list` or `list_something` return a [`ListPage`](/reference/class/ListPage) object,
containing properties `items`, `total`, `offset`, `count` and `limit`.
There are some exceptions though, like `list_keys` or `list_head` which paginate differently.
The results you're looking for are always stored under `items` and you can use the `limit`
property to get only a subset of results. Other properties can be available depending on the method.

```python
from apify_client import ApifyClient

apify_client = ApifyClient('MY-APIFY-TOKEN')

> # Resource clients accept an ID of the resource.

Reviewer: probably the only comment with dot in the end

```suggestion
# Resource clients accept an ID of the resource
```
